lib/flutter_secure_storage.dart

```diff
@@ -17,6 +17,12 @@ class FlutterSecureStorage {
     return value;
   }
 
+  Future<bool> contains({@required String key}) async {
+    final bool value =
+    await _channel.invokeMethod('contains', <String, String>{'key': key});
```

Comment:
What's about reusing "read" method and comparing with `null`?

Reply:
Using the 'read' method, if the key exists, is decrypted unnecessarily.